whatshap/cli/polyphase.py

```diff
@@ -62,6 +62,7 @@ def run_polyphase(
     block_cut_sensitivity=4,
     threads=1,
     use_prephasing=False,
+    no_mav=False,
```

Comment:
This would be nicer with `mav=True` because it avoids the double negative (see below)

Reply:
Fixed. See above.